lib/credo_naming/check/consistency/module_filename.ex

```diff
@@ -44,6 +44,7 @@ defmodule CredoNaming.Check.Consistency.ModuleFilename do
 
     source_file.filename
     |> String.starts_with?(excluded_paths)
+    |> Kernel.||(source_file.filename === "stdin")
```

Comment:
We should add a case in `test/credo_naming/check/consistency/module_filename_test.exs` to test this behavior.